src/main/java/com/redhat/devtools/lsp4ij/operations/documentation/LSPDocumentationProvider.java

```diff
@@ -70,15 +98,15 @@ public String getQuickNavigateInfo(PsiElement element, PsiElement originalElemen
     @Nullable
     @Override
     public String generateDoc(@NotNull PsiElement element, @Nullable PsiElement originalElement) {
-        if (!LanguageServersRegistry.getInstance().isLanguageSupported(originalElement.getContainingFile())) {
+        if (originalElement == null || !LanguageServersRegistry.getInstance().isLanguageSupported(originalElement.getContainingFile())) {
```

Comment:
I will remove this condition and move to line 118 since this hover provider is used for completion documentation.